apps/css-workshop/src/pages/transfer-list.astro

```diff
@@ -1,5 +1,7 @@
 ---
 import Layout from './_layout.astro';
+import Checkbox_ from '../components/Checkbox.astro';
+import Checkbox from './checkbox.astro';
```

Comment:
Looks like the checkbox page itself is being imported here

Reply:
resolved this on a new branch. pr soon